examples/one_degree_simulation.jl

```diff
@@ -117,15 +117,15 @@ function progress(sim)
     u, v, w = ocean.model.velocities
     T = ocean.model.tracers.T
     e = ocean.model.tracers.e
-    Tmin, Tmax = minimum(T), maximum(T)
+    Tmin, Tmax, Tavg = minimum(T), maximum(T), mean(T)
```

Comment:
does the `mean` ignore land? or do we need to do `compute!(Field(Average(T)))[1, 1, 1]`?

Reply:
Yes, all reductions of `Field` will ignore immersed cells

Reply:
A lot of code has previously used various workarounds, including using `interior`, because conditional reductions often did not compile on the GPU. But since our revamp of conditional operations, I have found that we can use simple clean syntax now.

We may also want something like `mean(view(T, :, :, Nz))` which would compute the surface average